zeq/event.h

```diff
@@ -55,11 +66,22 @@ class Event : public boost::noncopyable
     ZEQ_API flatbuffers::Parser& getParser();
     ZEQ_API const flatbuffers::Parser& getParser() const;
 
+#ifdef BOOST_NO_CXX11_RVALUE_REFERENCES
+    Event& operator=( Event& rhs )
+    {
+        _impl = rhs._impl;
+        const_cast<Event &>(rhs)._impl = 0;
```

Comment:
The correct declaration of an assignment operator is Event& operator=( const Event& rhs )